reference/orders.v2.oas2.yml

```diff
@@ -5189,6 +5234,56 @@ components:
           shipping_quotes:
             url: 'https://api.bigcommerce.com/stores/{$$.env.store_hash}/v2/orders/163/consignments/shipping/64/shipping_quotes'
             resource: /orders/163/consignments/shipping/64/shipping_quotes
+    shippingConsignment_Put:
+      allOf:
+        - type: object
+          properties:
+            id:
+              type: integer
+              example: 99
+              description: ID of shipping consignment to update.
+          required:
+            - id
+        - type: object
+          properties:
+            line_items:
+              type: array
+              minItems: 1
+              items:
+               type: array
+               items:
+                anyOf:
+                 - $ref: '#/components/schemas/orderCatalogProduct_Put'
+                 - $ref: '#/components/schemas/orderCatalogProduct_Post'
+                 - $ref: '#/components/schemas/orderCustomProduct_Post'
+                 - $ref: '#/components/schemas/orderCustomProduct_Put'
+      x-examples:
+        example-1:
+          line_items:
+            - id: 12
+              quantity: 2
```

Comment:
is this example for `shippingConsignment_Put` ? if it is then "id" is required

Reply:
Got rid of this example, as it was not being used, the example values are instead sourced from orderCatalogProduct or orderCustomProduct schemas